src/featureMatching/__tests__/__snapshots__/getBriefDescriptors.test.ts.snap

```diff
@@ -0,0 +1,2849 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`alphabet image, maxNbFeatures = 10 2`] = `
```

Comment:
Snapshot tests aren't very useful if we can't review them. Unless you don't care about the actual values and just want to prevent future regressions?

Reply:
I have indeed used them to prevent regressions, because it is code that is difficult to test otherwise.